### PR TITLE
incremental: only load dependnecyitems from last for unloaded files

### PR DESCRIPTION
**Pull request by ansyral**

### CODE REVIEW

**Comment by hellosnow** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Build.Engine/Incrementals/DependencyGraph.cs [hunk 1]
```diff
@@ -27,6 +27,13 @@ public sealed class DependencyGraph
             },
             new DependencyType
             {
+                Name = DependencyTypeName.UidInclude,
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Build.Engine/Incrementals/DependencyGraph.cs]: >UidInclude [](start = 42, length = 10)

rename to `OverwriteDocument`?

Comment (ansyral) [inline src/Microsoft.DocAsCode.Build.Engine/Incrementals/DependencyGraph.cs]: how about `overwrite`? a little shorter:)

**Comment by vicancy** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Build.Engine/Incrementals/DependencyGraph.cs (hunk 1 above)
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Build.Engine/Incrementals/DependencyGraph.cs]: Sound good!

---
In reply to: [96593710](https://github.com/dotnet/docfx/pull/1184#discussion_r96593710) [](ancestors = 96593710)

**Comment by vwxyzh** [comment]
:shipit:

**Pull request merged**